File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_33762.py
# SPDX-License-Identifier: Apache-2.0
import os
from flask import request, jsonify


def BenchmarkTest33762():
    json_value = (request.get_json(silent=True) or {}).get('payload', '')
    data = f'{json_value:.200s}'
    os.remove(str(data))
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest33762.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest33762 {

    @GetMapping("/BenchmarkTest33762")
    public void BenchmarkTest33762(@RequestParam("id") String id, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String userId = id != null ? id : "";
        StringBuilder bundle = new StringBuilder();
        bundle.append(userId);
        String data = bundle.toString();
        response.sendError(500, data);
    }
}
